src/main/sensors/barometer.c

```diff
@@ -37,6 +37,7 @@
 #include "drivers/bus_spi.h"
 #include "drivers/io.h"
 
+#include "drivers/time.h"
```

Comment:
Includes should be alphabetically ordered.

Reply:
In the absence of any dependency ordering that seems like a good idea. Again you could make a real project of that. Perhaps some automated cleanup of the codebase would be a good idea, even though it would introduce a scary number of changes?

Reply:
The point is that there should not be any dependency based ordering - every header file should stand for its own, if there are interdependencies then this is a bug.
Also, agreed to cleaning up / checking in a separate project. But that does not mean that until this is done, new additions / changes should not adhere to common practice.